doc/source/ray-overview/examples.rst

```diff
@@ -1088,7 +1088,7 @@ Ray Examples
         Logging Tune Runs to Comet ML.
 
     .. grid-item-card:: :bdg-success:`Tutorial`
-        :class-item: gallery-item serve
+        :class-item: gallery-item serving serve
```

Comment:
Seems like `serve` is not needed here?

```suggestion
        :class-item: gallery-item serving
```

Reply:
The library label `serve` is different from the old content label `serving`. In the future we will have filters for specific ray libraries, so IMO it makes sense to keep this and remove `serving` if anything.

Reply:
Oh I see, I did a quick scan through the page and totally missed the other ones tagged with `serve`.